packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -146,12 +145,18 @@ class AndroidCameraCameraX extends CameraPlatform {
     processCameraProvider!.unbindAll();
 
     // Configure Preview instance.
-    _resolutionPreset = resolutionPreset;
+    _resolutionPreset = mediaSettings?.resolutionPreset;
+
     final int targetRotation =
         _getTargetRotation(cameraDescription.sensorOrientation);
+
     final ResolutionInfo? previewTargetResolution =
-        _getTargetResolutionForPreview(resolutionPreset);
+        null != mediaSettings?.resolutionPreset
```

Comment:
`_getTargetResolutionForPreview` handles a null preset, so the extra logic isn't necessary.

Reply:
done.